fundingmanager.go

```diff
@@ -931,6 +837,114 @@ func (f *fundingManager) reservationCoordinator() {
 	}
 }
 
+// advanceFundingState will advance the channel fromt the markedOpen state to
+// the point where the channel is ready for operation. This includes sending
+// funding locked to the peer, adding the channel to the router graph, and
+// announcing the channel.
+func (f *fundingManager) advanceFundingState(
+	channel *channeldb.OpenChannel) error {
+
+	channelState, shortChanID, err := f.getChannelOpeningState(
+		&channel.FundingOutpoint)
+	if err == ErrChannelNotFound {
+		// Channel not in fundingManager's opening database,
+		// meaning it was successfully announced to the
+		// network.
+		return nil
```

Comment:
Do we also want to add that consistency check here? Basically, if the channel is open, then also check to ensure that it's been added to to the graph. Alternatively, it can just add it again, since it's a idempotent operation.

Reply:
Yeah, that could be a way to handle the missing graph edges indeed.

Reply:
:+1: can be done as a follow-up though.